### PR TITLE
display type evidence

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #75
Display the right colours and the correct pyramid logo depending on the type of evidence:
image

### CODE REVIEW

File: web/views/search_view.ex [hunk 1]
```diff
@@ -1,3 +1,24 @@
 defmodule Bep.SearchView do
   use Bep.Web, :view
+  def colour_evidence(type) do
```
Comment (SimonLab) [inline web/views/search_view.ex]: these two functions are similar, it's working at the moment but maybe there is a nicer way to do this

Comment (shouston3) [inline web/views/search_view.ex]: Maybe you could do something like this to dry the code:

```elixir
defp format_class(type, format_string) do
  cond do
    type in [1, 4, 9, 10, 11, 16, 18, 34] -> format_string.(1)
    type == 13 -> format_string.(2)
    type == 2 -> format_string.(3)
    type in [27, 14] -> format_string.(4)
    type in [5, 8, 22, 29, 30, 35] -> format_string.(5)
    true -> format_string.(0)
  end
end

defp colour_evidence_format(int) when int == 0, do: "b--white"
defp colour_evidence_format(int) do: "b--evidence-#{int}"

defp pyramid_logo_format(int) when int == 0, do: ""
defp pyramid_logo_format(int) do: "pyramid-grade#{int}.svg"

def colour_evidence(type), do: format_class(type, colour_evidence_format)
def pyramid_logo(type), do: format_class(type, pyramid_logo_format)
```

File: web/templates/search/results.html.eex [hunk 2]
```diff
@@ -9,7 +9,15 @@
   <h2 class="f6 pb2 bw1 bb bep-b--blue"> <%= @data["total"] %> Results</h2>
   <ul class="ph1 list">
     <%= for {evidence, i} <- Enum.with_index(@data["documents"]) do %>
```
Comment (shouston3) [inline web/templates/search/results.html.eex]: You could `Enum.with_index(@data["documents"], 1)` to offset the initial index by 1

Comment (SimonLab) [inline web/templates/search/results.html.eex]: nice :+1:

File: web/views/search_view.ex (hunk 1 above)
Comment (SimonLab) [inline web/views/search_view.ex]: yes this should work

**Pull request merged**